proxmoxer/backends/base_ssh.py

```diff
@@ -81,8 +81,8 @@ class JsonSimpleSerializer(object):
     def loads(self, response):
         try:
             return json.loads(response.content)
-        except ValueError:
-            return response.content
+        except (UnicodeDecodeError, ValueError, json.decoder.JSONDecodeError):
```

Comment:
I don't know if all of these are possible from the response from the shell API, but it is better to catch all possible exceptions that could result from an improper response